test pypi upload fails (using legacy?)

https://github.com/ETSInitiative/PETSIRD/blob/fd6b833d85169937d25e504f5e822571f1ace136/.github/workflows/test_pypi.yml#L48

Actions reports
```
Uploading petsird-0.0.1-py3-none-any.whl
25l
  0% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.0/43.8 kB • --:-- • ?
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.8/43.8 kB • 00:00 • 152.6 MB/s
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.8/43.8 kB • 00:00 • 152.6 MB/s
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.8/43.8 kB • 00:00 • 152.6 MB/s
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.8/43.8 kB • 00:00 • 152.6 MB/s
25hWARNING  Error during upload. Retry with the --verbose option for more details. 
ERROR    HTTPError: 400 Bad Request from https://test.pypi.org/legacy/          
         Bad Request                                                            
Error: Process completed with exit code 1.
```
@casperdcl @lukepolson suggestions?

--- a/python/pyproject.toml
+++ b/python/pyproject.toml
@@ -1,20 +1,43 @@
 [build-system]
-requires = ["setuptools>=42", "wheel"]
+requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "petsird"
-version = "0.0.1"  # Update as needed
+dynamic = ["version"]
 description = "Library and tools for working with PET Emission Tomography Standardization Initiative Raw Data (PETSIRD) data"
 readme = "README.md"
-authors = [{ name = "Kris Thielemans", email = "k.thielemans@ucl.ac.uk" }]
-license = { text = "Apache-2.0" }
+authors = [
+    # TODO
+    {name = "Kris Thielemans", email = "k.thielemans@ucl.ac.uk"},
+]
+maintainers = [
+    {name = "Casper da Costa-Luis", email = "imaging@cdcl.ml"},
+    {name = "Kris Thielemans", email = "k.thielemans@ucl.ac.uk"}]
+license = {text = "Apache-2.0"}
 requires-python = ">=3.9"
 dependencies = [
     "numpy>=1.22",
 ]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3 :: Only"]
+
+[tool.setuptools.packages.find]
+include = ["petsird", "petsird.*"]
+
+[tool.setuptools_scm]
+version_file = "petsird/version.py"
+root = ".."
 
 [project.urls]
-Homepage = "https://etsinitiative.org/"
-Repository = "https://github.com/ETSInitiative/PETSIRD.git"
+Homepage = "https://etsinitiative.org"
+Repository = "https://github.com/ETSInitiative/PETSIRD"
 Issues = "https://github.com/ETSInitiative/PETSIRD/issues"

